11/collection-meta.yaml

```
@@ -229,6 +229,9 @@ collections:
       reason_text: >
         The collection has
         L(unresolved sanity test failures, https://github.com/ansible-collections/google.cloud/issues/613).
+      updates:
+        - cancelled_version: 11.8.0
```

```suggestion
        - cancelled_version: 11.9.0
```

@felixfontein Now that 11.8.0 is out, should I change this to 11.9.0? After all, we didn't cancel the removal in 11.8.0.